libs/common-services/src/main/java/uk/gov/di/ipv/core/library/service/ConfigService.java

```diff
@@ -358,4 +367,15 @@ private <T> T getCriConfigForType(String connection, String criId, Class<T> conf
                             pathTemplate, e));
         }
     }
+
+    public Map<String, String> getCriConfigs() {
+        Map<String, String> configs = new HashMap<>();
+        for (var cri : CriIdentifer.values()) {
+            var criId = cri.getId();
+            var criConfig =
+                    getCriConfigForType(getActiveConnection(criId), criId, CriConfig.class, false);
```

Comment:
I'm a bit concerned that this only covers 'active' connections - what if the VC was generated by a (now inactive) connection.

You might need to use the SSM `getMultiple` to get all the connections for each CRI?